src/clr-angular/data/datagrid/enums/column-changes.enum.ts

```diff
@@ -6,4 +6,10 @@
 
 export enum DatagridColumnChanges {
   WIDTH,
+  HIDEABILITY,
 }
+
+export const ALL_COLUMN_CHANGES: DatagridColumnChanges[] = [
+  DatagridColumnChanges.WIDTH,
+  DatagridColumnChanges.HIDEABILITY,
```

Comment:
nit `HIDDEN`, this doesn't toggle the column's ability to be hidden or not.

Reply:
done